bip-emilengler-replacing-the-term-address.mediawiki

```diff
@@ -0,0 +1,46 @@
+<pre>
+  BIP: UNDEFINED
+  Titel: Replacing the term address
+  Author: Emil Engler <me@emilengler.com>
+  Status: Draft
+  Type: Informational
+  Created: 2019-10-17
+  License: CC0
+</pre>
+
+==Abstract==
+This BIP proposes a new term for 'address'
+
+==Specification==
+The new term is:
+''Bitcoin'' '''Invoice''' ''Address''
```

Comment:
Suggest making "Address" an extended temporary form only for backward compatibility purposes.

Reply:
Ok, will put into that section